taskt/Core/Metrics.cs

```diff
@@ -109,7 +110,7 @@ public class ExecutionMetric
     {
        public string FileName { get; set; }
        public TimeSpan AverageExecutionTime { get; set; }
-       public List<Core.Automation.Engine.ScriptFinishedEventArgs> ExecutionData { get; set; }
+       public List<Core.Automation.Engine.EngineEventArgs.ScriptFinishedEventArgs> ExecutionData { get; set; }
```

Comment:
I think you have used the `taskt.Core.Automation.Engine.EngineEventArgs` namespace above, you can simply use `ScriptFinishedEventArgs` for list type.

Reply:
This isn't a cleanup for the ExecutionMetric class. I just fixed anything that was broken as a result of the engine class changes. Miguel already removed the namespaces in his cleanup for Metrics

Reply:
Got it.